packages/core/src/common/registry/resolve-id-registry.ts

```diff
@@ -51,6 +55,14 @@ export default class ResolveIdRegistry<TType, TToken extends { [key: string]: un
             return matched.token;
         }
 
+        if (this._useFallback) {
```

Comment:
Hi @davidchin , I have added `defaultToken` in `_resolveToken`.

However, as we still keep the old `CreditCardPaymentStrategy` in `core` as the default payment strategy and base class for many strategies, I used a `useFallback` flag to control whether we have a default token.

All changes have been tested on my dev store. (To identify the modified SDK code, I added a custom console log.)

https://user-images.githubusercontent.com/88361607/204983770-4d400a0e-62af-4a01-8b87-4de301f7dba9.mov